manifests/t/TortoiseSVN/TortoiseSVN/1.14.29465/TortoiseSVN.TortoiseSVN.installer.yaml

```
@@ -0,0 +1,38 @@
+# Created with YamlCreate.ps1 v2.2.0 $debug=QUSU.CRLF.5-1-25206-1000.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: TortoiseSVN.TortoiseSVN
+PackageVersion: 1.14.29465
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerType: wix
+  InstallerUrl: https://osdn.net/frs/redir.php?m=auto&f=%2Fstorage%2Fg%2Ft%2Fto%2Ftortoisesvn%2F1.14.5%2FApplication%2FTortoiseSVN-1.14.5.29465-x64-svn-1.14.2.msi
```

```suggestion
  InstallerUrl: https://osdn.dl.osdn.net/storage/g/t/to/tortoisesvn/1.14.5/Application/TortoiseSVN-1.14.5.29465-x64-svn-1.14.2.msi
```
Better to use non-redirected URLs